Client/src/components/contanier/student/Coures/index.js

```diff
@@ -1,10 +1,24 @@
 import React, { Component } from 'react'
 
+import Course from '../../../common/Course'
+import Button from '../../../common/Button'
+import data from './staticData'
+import './index.css'
+
 export default class Courses extends Component {
```

Comment:
why you make it class if you will not use ``` state```  !!!

Reply:
I will use state after the fetch 
instead of the staticdata file